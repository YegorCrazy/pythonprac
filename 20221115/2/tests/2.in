class C:
	num = Num()

c = C()
print(c.num)
c.num = "aaaaa"
print(c.num)
c.num = 1.111
print(c.num)
