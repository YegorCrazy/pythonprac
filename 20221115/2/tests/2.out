0
5
1.111
